packages/ui-react/src/containers/Layout/Layout.tsx

```diff
@@ -38,13 +39,13 @@ const Layout = () => {
   );
   const isLoggedIn = !!useAccountStore(({ user }) => user);
   const favoritesEnabled = !!config.features?.favoritesList;
-  const { menu, assets, siteName, description, styling, features } = config;
+  const { menu, assets, siteName, description, features } = config;
   const metaDescription = description || t('default_description');
 
   const profileController = getModule(ProfileController, false);
 
   const { searchPlaylist } = features || {};
-  const { footerText } = styling || {};
+  const footerText = useMemo(() => unicodeToChar(env.APP_FOOTER_TEXT), []);
```

Comment:
This could also be placed outside the component (so we can ignore the `useMemo`)

Reply:
done